daemon/src/payout_curve.rs

```diff
@@ -721,13 +727,62 @@ mod tests {
         pretty_assertions::assert_eq!(actual_payouts, expected_payouts);
     }
 
+    #[test]
+    fn verify_effect_of_funding_fee() {
+        let payouts = calculate_payout_parameters(
+            Price::new(dec!(54000.00)).unwrap(),
+            Usd::new(dec!(3500.00)),
+            Leverage::new(5).unwrap(),
+            200,
+            0,
+        )
+        .unwrap();
+
+        let funding_fee = 100;
+        let payouts_with_fee = calculate_payout_parameters(
+            Price::new(dec!(54000.00)).unwrap(),
+            Usd::new(dec!(3500.00)),
+            Leverage::new(5).unwrap(),
+            200,
+            funding_fee,
+        )
+        .unwrap();
+
+        let sum_of_payouts_is_constant =
+            payouts
+                .iter()
+                .zip(payouts_with_fee.clone())
+                .all(|(payout, payout_with_fee)| {
+                    dbg!(payout.short_amount) + dbg!(payout.long_amount)
+                        == dbg!(payout_with_fee.short_amount) + dbg!(payout_with_fee.long_amount)
+                });
```

Comment:
Same here, I think it would be nice to map both to the same and then do a separate step where we assert that they are equal.

Reply:
Could even do that by collecting them both into `Vec` and use `assert_eq`.